docs: several changes to the ICS docs (#2163)
cosmos/interchain-security: docs/docs/validators/partial-set-security-for-validators.md

- `comission-rate` decimal in `[minRate, 1]` where `minRate` corresponds to the minimum commission rate set on the
provider chain (see `min_commission_rate` in `interchain-security-pd query staking params`).

If a validator does not set a commission rate on a consumer chain, the commission rate defaults to their commission rate on the provider chain.

Validators can set their commission rate even for consumer chains that they are not currently opted in on, and the commission rate will be applied when they opt in. This is particularly useful for Top N chains, where validators might be opted in automatically,
so validators can set the commission rate in advance.

If a validator opts out and then back in, this will *not* reset their commission rate back to the default. Instead, their
set commission rate still applies.

## Queries

> PSS introduces a number of queries to assist validators determine which consumer chains they have to validate, their commission rate per chain, etc.

Reviewer: ```suggestion
PSS introduces a number of queries to assist validators in determining which consumer chains they have to validate, their commission rate per chain, etc.
```

```
that can be used by validator with `provider-validator-address` address to retrieve the list of chains that it has to validate.

:::warning

> As a validator, the list of chains returned by `has-to-validate` is the list of chains they **should** be validating to avoid

Reviewer: ```suggestion
For a validator, the list of chains returned by `has-to-validate` is the list of chains the validator **should** be validating to avoid
```
